app/src/main/kotlin/com/simplemobiletools/notes/activities/MainActivity.kt

```diff
@@ -36,8 +36,12 @@ import java.io.File
 import java.nio.charset.Charset
 
 class MainActivity : SimpleActivity(), ViewPager.OnPageChangeListener {
+    private val STORAGE_OPEN_FILE_ACTION = 0
     private val STORAGE_OPEN_FILE = 1
     private val STORAGE_EXPORT_AS_FILE = 2
+
+    private var openFileFromAction: (() -> Unit)? = null
```

Comment:
this variable isnt needed, since everything is handled in this activity itself. Just call some functions directly.